subprocess/subprocess_windows.go

```diff
@@ -137,7 +137,12 @@ func (sub *Subprocess) CreateFrozen() (*SubprocessData, error) {
 	si.Flags = win32.STARTF_FORCEOFFFEEDBACK | syscall.STARTF_USESHOWWINDOW
 	si.ShowWindow = syscall.SW_SHOWMINNOACTIVE
 
-	useCreateProcessWithLogonW := !sub.NoJob || win32.IsWindows8OrGreater()
+	isWindows8OrGreater := win32.IsWindows8OrGreater()
+	useCreateProcessWithLogonW := !sub.NoJob || isWindows8OrGreater
+	creationFlags := win32.CREATE_SUSPENDED | syscall.CREATE_UNICODE_ENVIRONMENT
+	if !isWindows8OrGreater {
```

Comment:
I think this could stand a comment explaining the situation a little--as of Windows 8 nested job objects are allowed, and the task manager will create scheduled tasks in a job without `JOB_OBJECT_LIMIT_BREAKAWAY_OK`, so we can't use `CREATE_BREAKAWAY_FROM_JOB`, but don't need to because we can just create a nested job.